simple/dgram_waitset.c

```diff
@@ -84,6 +84,13 @@ static int send_recv()
 	struct fi_cq_entry comp;
 	int ret;
 
+	ret = fi_recv(ep, rx_buf, rx_size + ft_rx_prefix_size(), fi_mr_desc(mr),
```

Comment:
This is a subtlety that is very easy to overlook.  How do the other datagram based tests not fail as well?